test/visualTest.ts

```diff
@@ -50,675 +46,694 @@ import areColorsEqual = helpers.areColorsEqual;
 
-                    if (value.displayName) {
-                        expect(value.displayNameKey).toBeDefined();
-                    }
+      it("should render all of render with foreground color applied", (done) => {
+        visualBuilder.updateRenderTimeout(dataView, () => {
+          const words = <SVGElement[]>(visualBuilder.wordText && Array.from(visualBuilder.wordText));
 
-                    if (typeof value === "object") {
-                        objectsChecker(value);
-                    }
-                }
-            };
+          expect(isColorAppliedToElements(words, foregroundColor, "fill"));
 
-            objectsChecker(jsonData);
+          done();
         });
-    });
-
-    describe("Accessibility", () => {
-        describe("High contrast mode", () => {
-            const backgroundColor: string = "#000000";
-            const foregroundColor: string = "#ffff00";
+      });
 
-            beforeEach(() => {
-                visualBuilder.visualHost.colorPalette.isHighContrast = true;
-
-                visualBuilder.visualHost.colorPalette.background = { value: backgroundColor };
-                visualBuilder.visualHost.colorPalette.foreground = { value: foregroundColor };
-            });
+      function isColorAppliedToElements(
+        elements: SVGElement[],
+        color?: string,
+        colorStyleName: string = "fill"
+      ): boolean {
+        return elements.some((element: SVGElement) => {
+          const currentColor: string = element.style[colorStyleName];
 
-            it("should render all of render with foreground color applied", (done) => {
-                visualBuilder.updateRenderTimeout(dataView, () => {
-                    const words: JQuery<any>[] = visualBuilder.wordText.toArray().map($);
+          if (!currentColor || !color) {
+            return currentColor === color;
+          }
 
-                    expect(isColorAppliedToElements(words, foregroundColor, "fill"));
+          return areColorsEqual(currentColor, color);
+        });
+      }
+    });
+  });
 
-                    done();
-                });
-            });
+  describe("support highlight test", () => {
+    const elementCount: number = 74;
+    let dataViewWithHighLighted: DataView;
+    let hightlightedElementNumber: number;
 
-            function isColorAppliedToElements(
-                elements: JQuery[],
-                color?: string,
-                colorStyleName: string = "fill"
-            ): boolean {
-                return elements.some((element: JQuery) => {
-                    const currentColor: string = element.css(colorStyleName);
+    beforeEach(() => {
+      hightlightedElementNumber = Math.round(getRandomNumber(0, elementCount - 1));
 
-                    if (!currentColor || !color) {
-                        return currentColor === color;
-                    }
+      dataViewWithHighLighted = defaultDataViewBuilder.getDataView(undefined, true, hightlightedElementNumber);
```

Comment:
try calling visualBuilder.update() to eliminate timeout awaiting

Reply:
no luck